extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -164,25 +187,24 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   onLayoutMeasure() {
-    /** @private {number} */
     this.slideWidth_ = this.getLayoutWidth();
 
-    /** @private {number} */
     this.previousScrollLeft_ = this.slidesContainer_./*OK*/scrollLeft;
   }
 
   /** @override */
   layoutCallback() {
-    if (this.slideIndex_ == null) {
+    if (this.slideIndex_ === null) {
```

Comment:
my bad, its actually the reverse:

https://screenshot.googleplex.com/XZJWPZOvRLG.png

so probably on this rule:
`If Type(x) is Null, return true.`